classes/profile_table_page.php

```diff
@@ -61,6 +61,13 @@ public static function display(profile_table $table, \moodle_url $url): void {
                 $deletebutton->add_confirm_action(get_string('deleteallwarning', 'tool_excimer'));
                 echo $output->render($deletebutton);
             }
+            $filters = $table->get_filters();
+            if (count($filters)) {
```

Comment:
```suggestion
            if (!empty($filters)) {
```
Since the count itself is not important, we just want to know if there are filters applied at all or not.